cromwell-backend/src/main/scala/cromwell/backend/DefaultBackendFactory.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.backend
+
+import akka.actor.{ActorRef, ActorSystem, Props}
+import cromwell.backend.model.TaskDescriptor
+
+/**
+  * Returns a backend based on the type.
```

Comment:
I don't feel this comment adds anything the name doesn't already say. I'd either flesh it out or remove it